### ISSUE TITLE
Same module in nixpkgs - when to sync?

### ISSUE THREAD

**Issue by JohnAZoidberg** [nix-community/home-manager#713]
The compton module seems to be the same (with minor changes) as in nixpkgs. There it also users a systemd user service.
What's the policy for keeping them in sync (or not)?

I'm upgrading the package (nixos/nixpkgs#61681) but they have deprecated and changed some options. Should I make the same changes to this module once my PR lands in nixpkgs-unstable?
And even then I should probably have the new options depend on the package version because stable doesn't have that new package.

**Comment by rycee** [comment]
There is not really any explicit policy. Typically a package gets updated in Nixpkgs, breaks somebody's configuration when they do `home-manager switch`, and they then open a PR to fix it 🙂

To avoid the "breaking somebody's configuration" step I think it would be very nice to have these module updates in HM shortly after the package update getting into the Nixpkgs unstable channel.

Don't bother trying to adjust based on the package version, I'll just avoid cherry-picking the changes to the release-19.03 branch.

**Comment by JohnAZoidberg** [comment]
I'm also taking about the module. The Compton module seems pretty similar to the one in nixpkgs and they could benefit from updates to either one of them.

**Comment by rycee** [comment]
Yeah, I was also referring to the module. The module would need to be updated in both NixOS and Home Manager.

**Comment by teto** [comment]
hopefully nix flakes would help with that https://gist.github.com/edolstra/40da6e3a4d4ee8fd019395365e0772e7 . We would need a way to change the out path of the generated config though
